src/api/customprotocol-core/stream.ts

```diff
@@ -385,6 +385,8 @@ export class Stream<H extends HandlerTypes = {}> extends EventEmitter<Union<Hand
     return false;
   }
 
+  // TODO: fix inheritance - _isFirst is used in Stream implementation so can't be removed here
```

Comment:
Feels like its not an issue of inheritance, more like an issue of linter. Would that work if you replaced `_isFirst` with `_` ?

Reply:
We can `_` but then the next person landing on this code will ask himself why there is an unused variable, will likely remove it, and broke again the inheritance. So we should keep the comment to explain why there is an unused variable here until we fix the inheritance, a.k.a we don't add unused variable because parents might use them.